forge/test/model_demos/high_prio/cnn/pytorch/test_detr.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+# SPDX-License-Identifier: Apache-2.0
+# Detr model having both object detection and segmentation model
+# https://huggingface.co/docs/transformers/en/model_doc/detr
+
+import pytest
+import forge
+import requests
+from torchvision import transforms
+from transformers import (
+    DetrForObjectDetection,
+    DetrImageProcessor,
+    DetrConfig,
+    DetrFeatureExtractor,
+    DetrForSegmentation,
+)
+from PIL import Image
+
+detection_variants = ["facebook/detr-resnet-50"]
+segmentation_variants = ["facebook/detr-resnet-50-panoptic"]
+
+# Load the image
+url = "http://images.cocodataset.org/val2017/000000397133.jpg"
+image = Image.open(requests.get(url, stream=True).raw).convert("RGB")
+
+
+@pytest.mark.parametrize("variant", detection_variants)
+def test_detr_detection(variant):
+
+    # Load the model
+    model = DetrForObjectDetection.from_pretrained(variant)
+    processor = DetrImageProcessor.from_pretrained(variant)
+    model_config = DetrConfig.from_pretrained(variant)
+
+    # Preprocess the image for the model
+    inputs = processor(images=image, return_tensors="pt")
```

Comment:
It seems that `inputs` are unused. is there a reason for inputs to exist?
Also on a side note, is there a difference between this and `transforms.ToTensor()(image)`?
We probably should keep one of them and discard the other one :)